projects/batfish/src/org/batfish/grammar/cisco/Cisco_mpls.g4

```diff
@@ -80,6 +80,25 @@ null_mplst_stanza
       | AUTO_TUNNEL
       | LOGGING
       | REOPTIMIZE
+      | reoptimize_stanza
```

Comment:
This really doesn't belong here. The only reason *_null_stanza rules exist is to make it easy to ignore a bunch of things based on the first word of the  line. It is not to mark it as being ignored. If you are going to make a structured thing to ignore, just put the rule with all the other functional ones and mark it as unimplemented in the listener function for the rule. Furthermore, this rule does NOTHING because it is right under REOPTIMIZE which is also the first token in its definition.